spring-cloud-gcp-samples/pom.xml

```diff
@@ -121,6 +122,13 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
+				<groupId>com.google.cloud</groupId>
+				<artifactId>spring-cloud-gcp-dependencies</artifactId>
+				<version>${project.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
 		</dependencies>
```

Comment:
We probably do not want to add `spring-cloud-gcp-dependencies` here to the `spring-cloud-gcp-samples/pom.xml`.
We have this section added to each individual samples pom.xml (e.g. [spanner sample](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/0ecd59a652476ce08ee1192bca9238b4d43514e2/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/pom.xml#L17-L28)) to demonstrate the usage of BOM to in user application, and to keep each sample able to work independently. Thus `spring-cloud-gcp-samples/pom.xml` contains mostly setups needed to run ci or build.

Reply:
makes sense, remove it from `spring-cloud-gcp-samples/pom.xml`